<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>587</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/illust.jpg);</string>
  </property>
  <widget class="QPushButton" name="signup">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>Click Here to Sign up</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>243</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;
background-image: url(:/background.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>243</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="id">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>ID：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="idEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;
background-image: url(:/background.png);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="stu">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>I'm a Student</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tea">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>I'm a Teacher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="admin">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>I'm an administer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>95</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ok_cancel">
    <item>
     <widget class="QPushButton" name="ok_Button">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_Button">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_Button</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
